contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -71,7 +67,6 @@ contract SmartAccount is
         owner = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
```

Comment:
Are we using this variable in our contract?

mapping(bytes32 => uint256) public signedMessages;

Reply:
yep, it's used to mark approved messages as part ofEIP1271 flow